vmdb/spec/models/ems_refresh/parsers/foreman_spec.rb

```diff
@@ -79,7 +84,8 @@
         "last_compile"       => date1,
         "location_id"        => 20,
         "organization_id"    => 10,
-        "build"              => true
+        "puppet_status"      => 0,
```

Comment:
Also, here's it's an integer, so if it gets set into a String column we have to be sure to .to_s it.